src/test/java/io/gravitee/policy/apikey/ApiKeyPolicyIntegrationTest.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.policy.apikey;
+
+import io.gravitee.apim.gateway.tests.sdk.configuration.GatewayConfigurationBuilder;
+import io.gravitee.definition.model.Api;
+import io.gravitee.definition.model.ExecutionMode;
+import io.gravitee.policy.v3.apikey.ApiKeyPolicyV3IntegrationTest;
+
+/**
+ * @author Jeoffrey HAEYAERT (jeoffrey.haeyaert at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public class ApiKeyPolicyIntegrationTest extends ApiKeyPolicyV3IntegrationTest {
```

Comment:
```suggestion
public class ApiKeyPolicyFullJupiterIntegrationTest extends ApiKeyPolicyV3IntegrationTest {
```

Reply:
As a naming convention we will choose to name the integration test as follow:

- `ApiKeyPolicyIntegrationTest`: full Jupiter mode
- `ApiKeyPolicyV3CompatibilityIntegrationTest`: Jupiter mode with V3 execution compatibility
- `ApiKeyPolicyV3IntegrationTest`: full v3, no Jupiter at all